Source/FlowEditor/Public/UnrealExtensions/IFlowExtendedPropertyTypeCustomization.h

```diff
@@ -63,7 +63,8 @@ StructT* IFlowExtendedPropertyTypeCustomization::TryGetTypedStructValue(const TS
 		TArray<void*> RawData;
 		StructPropertyHandle->AccessRawData(RawData);
 
-		if (RawData.Num() > 0)
+		// (must be exactly one, multi-select is not supported for this feature)
```

Comment:
multiselect was a big problem with the curated name list.  so we're just punting on it.  if you multiselect, it just doesn't show you the drop-down at all.